### ISSUE TITLE
HTTPS on facepunch.github.io/webrcon/

### ISSUE THREAD

**Issue by githtz** [Facepunch/webrcon#26]
When using HTTPS on facepunch.github.io/webrcon/ does not allow to connect to server. 
The error message looks like this:
`Error: The operation is insecure.
RconService/Service.Connect@https://facepunch.github.io/webrcon/js/rconService.js:19:19
ConnectionController/$scope.Connect@https://facepunch.github.io/webrcon/js/connection.js:62:3
fn@https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js line 213 > Function:4:209
e@https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js:254:74
$eval@https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js:133:309
$apply@https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js:134:12
compile/</<@https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js:254:124
dispatch@https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:4:8497
add/r.handle@https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:4:5235`

**Comment by bwollermann** [comment]
Yeah whoever manages the site must not care to keep a valid cert on their site.  Just go to the HTTP link and it will work just fine assuming you have the correct RCON info.

http://facepunch.github.io/webrcon/#/home
